pkg/util/gziphandler/gzip.go

```diff
@@ -158,6 +158,10 @@ func (w *GzipResponseWriter) Write(b []byte) (int, error) {
 	return w.startPlainWrite(len(b))
 }
 
+func (w *GzipResponseWriter) Unwrap() http.ResponseWriter {
```

Comment:
Can you explain why is this change required?

Reply:
I've added a comment that explains why this is required.